enhancements/sig-policy/28-selective-policy-enforcment/README.md

```diff
@@ -0,0 +1,624 @@
+fleet.
+
+With Selective Policy Enforcement a feature is introduced which allows
+the user control over the timing of the "enforce" remediationAction
+taking effect on a selected subset of the bound clusters.
+
+## Motivation
+
+Selective Policy Enforcement allows fine grained control over the
+timing and application of a policy to clusters. It is intended to
+support this level of control natively within the policy framework. By
+supporting this as a feature users, or higher level
+controllers/orchestrators, do not have to implement a procedural
+pattern of copying policies and manipulating bindings/labels to
+progressivly enforce the policy on clusters.
+
+*Progressive Policy Rollout*
+
+One of the key results of this feature is allowing the impact of a
+Configuration Policy change to be "rolled out" to a fleet of clusters
+in a progressive way rather than all at once. The timing and choice of
+clusters to which the change is applied are typically use case
+dependent but this feature allows scenarios such as
++ Application to a small set of "soak" clusters before rolling the
+  change out to the entire fleet in progressively larger groups
++ Staggered updates to overlapping (geographical or logical) clusters
+  to ensure continuous service coverage
++ Meeting Service Level Agreements for availability by limiting
+  concurrent updates
++ Applying changes to groups of clusters with varying "maintenance
+  windows" when SLAs allow changes to be made
++ Applying changes to one region while deferring other regions
+
+### Goals
+
+The goals of this feature are to
+1. Enable users to select when a policy will be enforced
+1. Enable users to select what subset of bound clusters the policy
+   will be enforced to
+1. Support progressive policy rollouts as described above
+1. Give users visibility into the effect of a Configuration Policy
+   change prior to remediation
+1. Reduce complexity and scalability issues with alternative
+   approaches
+
+### Non-Goals
+
+Out of scope for this feature
+1. A mechanism/feature to capture/define the timing of enforcement
+1. A feature for managing progressive rollouts. The timing and cluster
+   selection are typically use-case specific. There may be enough
+   commonality to define a feature to manage progressive rollouts but
+   that is deferred to a separate discussion/enhancement.
+
+## Proposal
+
+The Selective Policy Enforcement feature extends the policy APIs to
+allow enforcement of inform policies to selected clusters. The effect
+of this would be the equivalent of setting the remediationAction of
+child (cluster namespace copy) policies to enforce for those selected
+clusters.
+
+The proposal extends the API with `remediationActionOverride` and
+includes two configurable elements as shown in this sample:
+
+```
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+...
+remediationActionOverride:
+    remediationAction: [null|enforce]
+    subFilter: [false|true]
```

Comment:
Does this field belong to the binding itself? I thought this was for the binding part, not so much for the action override?

Reply:
I think it stays here with the override. We don't want to change the general semantics of the binding or of multiple bindings. The current behavior of multiple bindings is that they associate a policy with clusters and multiple bindings are a "union". For this feature, and our use case, the subFilter is a mechanism to select a subset of the already bound clusters and do the override to them.

Reply:
Actually, I think it might make sense to have it at the root level since it would allow us to add additional overrides in the future but have a `subFilter` value per `PlacementBinding`.

Reply:
I've updated the proposal to move the subFilter to the root level so that it is a control on how the binding itself works. As @jc-rh recommended this makes the override control simpler (it applies to this whole binding) and allows future enhancements to benefit from subFiltering. Thanks!